### PR TITLE
Fix stringify of error object

### PR DESCRIPTION
**Pull request by SukantGujar**
JSON.stringify doesn't correctly process `Error` instances, unless explicit keys are provided. See http://stackoverflow.com/questions/18391212/is-it-not-possible-to-stringify-an-error-using-json-stringify#comment57014279_26199752.

### CODE REVIEW

**Comment by fahad19** [comment]
nice catch!

@SukantGujar: would you like to be a maintainer of this package? I can add you to the repo and on npm then. you can then proceed with the release.

/cc @JustinLivi

**Comment by SukantGujar** [comment]
Sure, thanks @fahad19.

**Comment by fahad19** [comment]
@SukantGujar: what's your npm username?

**Comment by SukantGujar** [comment]
It's sukantgujar

**Comment by fahad19** [comment]
You have been added to both the repo and on npm.

please follow the maintainers guide for proceeding with a release, after this PR has been merged.

and welcome!

**Review by fahad19 — APPROVED** [review]

**Comment by SukantGujar** [comment]
Thanks @fahad19

**Pull request merged**
